# Compilation fails with:
#mtl_ofi_component.c:298:5: error: call to undeclared function 'mca_common_cuda_fini'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
#    mca_common_cuda_fini();
#    ^
#1 error generated.
# Patch generated from OpenMPI version 4.1.5
# April 25 2024 by B. Hajgato (UGent)
--- openmpi-4.1.4/ompi/mca/mtl/ofi/mtl_ofi_component.c	2022-05-26 16:32:35.000000000 +0200
+++ openmpi-4.1.5/ompi/mca/mtl/ofi/mtl_ofi_component.c	2023-02-23 05:25:04.000000000 +0100
@@ -294,9 +294,6 @@
 static int
 ompi_mtl_ofi_component_close(void)
 {
-#if OPAL_CUDA_SUPPORT
-    mca_common_cuda_fini();
-#endif
     return opal_common_ofi_close();
 }
 
